### DISCUSSION TITLE
500 Internal Server Error

### DISCUSSION THREAD

**Discussion by itsAkshayDubey** [community/community#18940] [Other Feature Feedback, Questions, & Ideas] [answered]
Unable to push code using CLI, getting below error:

![image](https://user-images.githubusercontent.com/38462415/174511745-bb9f1694-8295-4417-92a1-50f8b4072c97.png)

When trying to edit and commit the code directly on GitHub - getting HTTP-500, as below

![image](https://user-images.githubusercontent.com/38462415/174512067-d7bb36e4-9f97-43a6-84e7-dda8d74f0696.png)

Facing this issue only with a particular repository since yesterday.

Any solution would be greatly appreciated. Thanks.

**Comment by melvincwng** [comment]
Hi @itsAkshayDubey, seems like your local repository is not in sync with the remote code repository. 

Try to do the following:
1. git pull origin master
2. git push origin master

Hopefully this helps. Otherwise, consider creating a post about your issue in Github Community Forums here https://github.community/ so that others can help you too.

If this is helpful, please kindly 'Mark as answer' so that others know this issue has been answered.

Thanks 😊

**Reply by itsAkshayDubey** [reply]
Hello @melvincwng ,

I tried this, but unfortunately still getting the same error.
=> Used _Hamiltonian-cycle-implementation_ instead of _master_ as it is the branch where I need to push the changes.

![image](https://user-images.githubusercontent.com/38462415/174519784-e94f2507-c928-4491-b80e-2c24b24b9339.png)

Also tried _--force_ push, but still no luck.

![image](https://user-images.githubusercontent.com/38462415/174520066-8e43886a-1a85-4230-89ab-35c75860f217.png)



> @itsAkshayDubey, if the above doesn't work, as a LAST resort, you can consider trying this:
> 
>     1. git push origin master --force
> 
> 
> This command essentially force pushes the changes in your local repository to your remote repository. However, you might want to read up more about this command from [this link](https://stackoverflow.com/questions/33247309/why-is-it-dangerous-to-do-a-force-push-against-a-remote-repository) first before trying it out. As it is dangerous to do a force push to your remote repository if you do not know what you are doing.
> 
> Hope this resolves your issue.

**Comment by melvincwng** [comment]
@itsAkshayDubey, if the above doesn't work, as a LAST resort, you can consider trying this:

1. git push origin master --force

This command essentially force pushes the changes in your local repository to your remote repository. However, you might want to read up more about this command from [this link](https://stackoverflow.com/questions/33247309/why-is-it-dangerous-to-do-a-force-push-against-a-remote-repository) first before trying it out. As it is dangerous to do a force push to your remote repository if you do not know what you are doing.

Hope this resolves your issue.

**Reply by itsAkshayDubey** [reply]
> @itsAkshayDubey, if the above doesn't work, as a LAST resort, you can consider trying this:
> 
>     1. git push origin master --force
> 
> 
> This command essentially force pushes the changes in your local repository to your remote repository. However, you might want to read up more about this command from [this link](https://stackoverflow.com/questions/33247309/why-is-it-dangerous-to-do-a-force-push-against-a-remote-repository) first before trying it out. As it is dangerous to do a force push to your remote repository if you do not know what you are doing.
> 
> Hope this resolves your issue.

@melvincwng 
Unfortunately no luck with this one as well...

**Comment by melvincwng** [comment, marked as answer]
@itsAkshayDubey, try posting your issues in Github Community Forums to see if anyone else has an answer.

Alternatively, if needed, you can try raising a support ticket at Github support.

All the best
